airflow/providers/qubole/hooks/qubole.py

```diff
@@ -103,6 +103,8 @@ def build_command_args() -> Tuple[Dict[str, list], list]:
     return command_args, list(hyphen_args)
 
 
+COMMAND_ARGS: List[str]
+HYPHEN_ARGS: List[str]
```

Comment:
We need to explicitly create these variables so that sphinx-autoapi can see and import these variables in other module.